org/pr/milestone.ts

```diff
@@ -1,19 +1,32 @@
 import {warn, danger} from "danger";
 
 export default async () => {
-    // Skip for release PRs
     const githubLabels = danger.github.issue.labels;
+    const currentPR = await danger.github.api.pulls.get(danger.github.thisPR);
+
+    // Skip for draft PRs
+    if (currentPR.data.draft) {
+        return;
+    }
```

Comment:
Here's something I discovered while testing this against real PR.

If a PR was a draft and then it's closed, `currentPR.data.draft` won't be `null`/`false` and we'll hit the `return` here.

Obviously not an issue, because we don't care about running these checks on closed PRs, but I still found interesting. My guess would be that GitHub doesn't remove that state information when closing the PR, which makes sense.